css/style.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 10px;
+  display: block;
+}
+
+body {
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+}
+
+#title {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+section {
+  display: block;
+}
+
+#headline {
+  margin: 50px 0;
+  text-align: center;
+  font-weight: bold;
+  font-size: 1.17em;
+}
+
+ul {
+  max-width: 550px;
+  margin: 0 auto 50px auto;
+  text-align: left;
+  line-height: 1.6;
+  display: block;
+}
+li {
+  margin: 16px 0;
+}
```

Comment:
[optional] you can select those elements by giving them a class for nothing else but best practice